pom.xml

```diff
@@ -171,6 +170,30 @@
         <version>1.2</version>
       </plugin>
     </plugins>
+    <pluginManagement>
+      <plugins>
```

Comment:
there is a site distribution URL so may also need the site plugin and associated bits.